selfdrive/car/hyundai/interface.py

```diff
@@ -45,8 +45,13 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
         if candidate not in CANFD_RADAR_SCC_CAR:
           ret.flags |= HyundaiFlags.CANFD_CAMERA_SCC.value
     else:
+      # detect non-HDA2 with CAN and CAN-FD definitions
```

Comment:
I think it makes sense to add them to CANFD list, and dynamically detect if they are using CAN messages by message presence